examples/communication/ip/SendIPDataSample/ReadMe.txt

```diff
@@ -0,0 +1,89 @@
+  Introduction
+  ------------
+  This sample Java application shows how to send IP data from an IP device to 
+  another one connected to the Internet using the XBee Java Library.
+
+  The application sends IP data to another Wi-Fi device on the network with a 
+  specific IP address and port number.
+
+  NOTE: This example uses the Wi-Fi device (WiFiDevice) class, but it can be 
+        applied to other Internet capable XBee device classes such as 
+        CellularDevice.
+
+
+  Files
+  -----
+    * com.digi.xbee.api.sendipdata.MainApp.java:
+      Main application class. It instantiates a Wi-Fi device, establishes a 
+      serial connection with it and sends the IP data to the XBee device with 
+      the IP address and port specified. Finally it prints out the result of the 
+      sent operation.
+
+
+  Requirements
+  ------------
+  To run this example you will need:
+
+    * At least two Internet capable XBee radios (for example Wi-Fi radios) in 
+      API mode and their corresponding carrier boards (XBIB or equivalent).
+    * The XCTU application (available at www.digi.com/xctu).
+
+
+  Example setup
+  -------------
+    1) Plug the XBee radios into the XBee adapters and connect them to your
+       computer's USB or serial ports.
+
+    2) Ensure that the modules are in API mode and connected to the same access
+       point.
+       For further information on how to perform this task, read the 
+       'Configuring Your XBee Modules' topic of the Getting Started guide.
+
+    3) Set the port and baud rate of the sender (local) XBee radio in the 
+       MainApp class.
+       If you configured the modules in the previous step with the XCTU, you 
+       will see the port number and baud rate in the 'Port' label of the device 
+       on the left view.
+
+    4) Set the destination IP address and port number in the MainApp class.
+       You can find them by reading the 'MY' and 'C0' settings of the 
+       destination XBee device with XCTU.
+       Note that the value of the 'C0' setting has hexadecimal format, so you 
+       have to convert it to decimal before setting it in the MainApp class.
+
+
+  Running the example
+  -------------------
+  First, build the application. Then, you need to set up XCTU to see the data 
+  received by the sender XBee device. Follow these steps to do so:
```

Comment:
I would change 'received' by 'sent' here or rephrase it to "... to see the data received by the remote XBee device."